2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,18 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  mealArray.sort((a, b) => a.price - b.price);
```

Comment:
Really good. You can move the sort() method lower down. No point sorting array unless it's length > 1